decentraland/starter.js

```diff
@@ -1,6 +1,22 @@
 /* eslint no-console: "off" */
-const { exec } = require("child_process");
-const { checkBlockchain } = require("./helpers.js");
+import { exec } from "child_process";
+
+// Note: Copied from helpers.js because import isn't working
+//import { checkBlockchain } from "./helpers";
```

Comment:
Interesting

Reply:
Yeah, I've explained why on the thread https://github.com/tasitlabs/tasit/pull/207#discussion_r270443373

Reply:
Yep, I saw that after making this comment. Resolving this